src/process_utility.c

```diff
@@ -728,8 +757,16 @@ process_vacuum(ProcessUtilityArgs *args)
 					if (hypertable_is_distributed(ht))
 						continue;
 
-					ctx.ht_vacuum_rel = vacuum_rel;
-					foreach_chunk(ht, add_chunk_to_vacuum, &ctx);
+					if (ht->fd.compressed == true)
```

Comment:
nit: `if (ht->fd.compressed)`

Reply:
@pmwkaa This nit is a matter of taste. I can see that @erimatnor changes conditions of your suggestion into explicit conditions in https://github.com/timescale/timescaledb/pull/2625

Reply:
If it matters, I have a habit of using explicit conditions for pointers, since that makes it clear that one is checking a pointer as opposed to a boolean. I have rarely, if ever, used explicit conditions for booleans.